src/buildercore/core.py

```diff
@@ -218,15 +218,13 @@ def find_rds_instances(stackname, state='available'):
 
     except botocore.exceptions.ClientError as err:
         LOG.info(err.response)
-        msg = err.response['Error']['Message']
         invalid_dbid = "Invalid database identifier"
```

Comment:
could be renamed to `invalid_dbid_message_prefix` or similar, since the value of this variable is not a db ID